core/sup_tree_core/executor_pool/async_job_runner.ex

```diff
@@ -218,14 +219,15 @@ defmodule AntikytheraCore.ExecutorPool.AsyncJobRunner do
 
   defp write_failed_log(
          %{
-           context: %Context{context_id: context_id},
+           context: %Context{gear_name: gear_name, context_id: context_id},
            logger_name: logger_name,
            log_prefix: log_prefix
          },
          end_time,
          error_reason
        ) do
     Writer.error(logger_name, end_time, context_id, log_prefix <> error_reason)
+    L.error("Process killed: gear_name=#{gear_name}, context_id=#{context_id}")
```

Comment:
This function is used by `:DOWN` other than `:kill`, so you should use `error_reason` instead of the fixed `killed` message.
https://github.com/access-company/antikythera/blob/a0d287545f8f80ad4c94dfcc9beb7083ad94682f/core/sup_tree_core/executor_pool/async_job_runner.ex#L167

Reply:
I changed the logging location. Antikythera operators only want to know process killed, not all of process down reason.
https://github.com/access-company/antikythera/pull/157/commits/d46becf17384ce214ed1f750d5c73c7050c28ef3